_stories/anomaly-detection.md

```diff
@@ -8,6 +8,7 @@ tags:
 persona:
 stage:
 company:
+order: 1
```

Comment:
@tracom1 the module is in the latest version of this branch. Have you tested this out yet? I think we just need to get the stories in the order that you'd like, merge to staging, and you'll have a nice candidate to present to TAC.